test/render.test.js

```diff
@@ -101,6 +107,12 @@ describe('snippet', function() {
         'analytics.load("key")');
     });
 
+    it('should set the load options', function() {
+      assertContains(
+        snippet.max({ apiKey: 'key', load: { integrations: { All: false } } }),
+        'analytics.load("key", {"integrations":{"All":false}})');
+    });
+
```

Comment:
seems like this test is a dupe from the one a few lines above?

Reply:
Yep, the first is for the `describe('#max', ... )` block and the second for the `describe('#min', ...)`. I noticed all other tests were done this way so just followed that same pattern. Happy to do it a different way if you have a recommendation.